packages/derive/src/lib.rs

```diff
@@ -98,7 +98,7 @@ impl Parse for Options {
 ///
 /// The VM will use this as a hint whether it needs to run the migrate function of your contract or not.
 ///
-/// ```
+/// ```no_run
```

Comment:
Doesn't `no_run` still compile the test or am I misremembering?

Reply:
Yes, you are right. Interestingly, it still fixes the problem for me. Maybe it's not a full compile? The error seems to be coming from LLVM, so if it only compiles to MIR and does type-checking, then it probably doesn't encounter it.

Reply:
Huh, interesting. Okay. I mean if it fixes it *and* keeps type-checking then that's a nice bonus.